scalameta/dialects/shared/src/main/scala/scala/meta/Dialect.scala

```diff
@@ -78,7 +78,8 @@ final class Dialect private (
     val allowExtensionMethods: Boolean,
     // Open modifier for classes introduced in dotty
     val allowOpenClass: Boolean,
-    // Whitebox macros introduced in dotty (splices/quotes)
+    // Scala 3 splices/quotes
+    @deprecated("Replaced with allowSplicesAndQuotes", "4.4.1")
     val allowWhiteboxMacro: Boolean,
```

Comment:
I think you can turn this into `def allowWhiteboxMacro = allowSpliceAndQuote`

Reply:
`allowWhiteboxMacro` no longer is used, so I think there is no need to do it.